main.js

```diff
@@ -0,0 +1,28 @@
+alert ('Lets play rock paper scissors!!');
+var valuePick = prompt('Pick rock, paper or scissors');
+//<------------------ Player picks number and it is set to valuePick
+var computerChoices = ['rock','paper', 'scissors']
+var computerChoice = Math.floor(Math.random() * computerChoices.length)
+  console.log(computerChoice);
+//<--------------------Computer picks array value at random
+if (valuePick === computerChoices[computerChoice]){
+    alert ('draw');
+
+} else if (valuePick === 'rock' && computerChoices[1]){
+  alert('Computer picks paper! Computer Wins!')
+
+} else { //(valuePick === 'rock' && computerChoices[2])
```

Comment:
when you write else, you're saying "If anything else other than the conditions I've specified happen" - you don't specify any conditions for else.